inc/gateways/class-base-stripe-gateway.php

```
@@ -181,6 +252,313 @@ public function setup_api_keys($id = false): void {
+			'country'       => 'US', // Could be made dynamic based on site settings
+		];
+	}
+
+	/**
+	 * Generate Stripe Connect OAuth authorization URL via proxy server.
+	 *
+	 * @since 2.x.x
+	 * @param string $state CSRF protection state parameter (unused, kept for compatibility).
+	 * @return string
+	 */
+	public function get_connect_authorization_url(string $state = ''): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$state'. (undefined)

(UnusedFormalParameter)